frontend/public/kubevirt/components/vm.jsx

```diff
@@ -330,6 +230,56 @@ const mapDispatchToProps = () => ({
   watchK8sList: actions.watchK8sList,
 });
 
+const ConnectedVmDetails = ({ obj: vm }) => {
+  const vmResource = getResourceKind(VirtualMachineModel, vm.metadata.name, true, vm.metadata.namespace, false);
+  const vmiResources = getResourceKind(VirtualMachineInstanceModel, vm.metadata.name, true, vm.metadata.namespace, true, getLabelMatcher(vm));
+  const podResources = getResourceKind(PodModel, undefined, true, vm.metadata.namespace, true, getLabelMatcher(vm));
+  const migrationResources = getResourceKind(VirtualMachineInstanceMigrationModel, undefined, true, vm.metadata.namespace, false); // https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachineinstancemigration
+  const resources = [vmResource, vmiResources, podResources, migrationResources];
+
+  return (
+    <Firehose resources={resources} flatten={getFlattenForKind(VirtualMachineModel.kind)}>
```

Comment:
I would remove the flatten function as I do not find it useful when watching multiple resources